### DISCUSSION TITLE
posts/Go/gin_swagger/

### DISCUSSION THREAD

**Discussion by Q1mi** [Q1mi/BlogComments#331] [Announcements]
https://www.liwenzhou.com/posts/Go/gin_swagger/ 

 李文周的Blog gin swagger gin_swagger 自动生成 接口 文档

**Comment by GGGIsGGG** [comment]
七米大佬，我是从go课程过来的。我的配置基本都是照抄的。但是不知道为何，下面Models里面死活没有models.ParamPostList 这个struct出现，只有其他两个，但是接口try it out那里有正确的请求参数位置可填的。扫了遍文档都没找到原因。

**Comment by dxgzg** [comment]
在go1高级版本之后，go get没有编译生成二进制可执行文件了，应该使用install

**Comment by ginginT** [comment]
cookie 可以设置吗

**Comment by highColdBoy** [comment]
多文件上传代码有一处有误 files := form.File["file"] --> files := form.File["file[]"]
